concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-1209	1209	schwach	schwach	weak	1601	WEAK
+Steinthal-1875-1565-1210	1210	früh	früh	early	672	EARLY
+Steinthal-1875-1565-1211	1211	spät	spät	late	477	LATE
+Steinthal-1875-1565-1212	1212	Er übertrifft dich an Grösse (ist grösser als du).	Er übertrifft dich an Grösse (ist grösser als du).	He is taller than you.		
+Steinthal-1875-1565-1213	1213	Sie übertreffen euch an Zahl (sind zahlreicher als ihr).	Sie übertreffen euch an Zahl (sind zahlreicher als ihr).	They are more than you.		
+Steinthal-1875-1565-1214	1214	Ich bin stärker als du.	Ich bin stärker als du.	I am stronger than you.		
+Steinthal-1875-1565-1215	1215	Er ist ausgezeichnet an Grösse, Schönheit.	Er ist ausgezeichnet an Grösse.	He is outstandingly tall.		
+Steinthal-1875-1565-1216	1216	Er ist ausgezeichnet an Grösse, Schönheit.	Er ist ausgezeichnet an Schönheit.	He is outstandingly beautiful.		
+Steinthal-1875-1565-1217	1217	heiter	heiter	cheerful	1976	MERRY
+Steinthal-1875-1565-1218	1218	ernst	ernst	serious		
+Steinthal-1875-1565-1219	1219	lustig	lustig	funny	1976	MERRY
+Steinthal-1875-1565-1220	1220	betrübt	betrübt	sad	699	SAD
+Steinthal-1875-1565-1221	1221	Er betrauert seinen Vater.	Er betrauert seinen Vater.	He mourns his father.		
+Steinthal-1875-1565-1222	1222	Ich bedaure dich.	Ich bedaure dich.	I pity you.		
+Steinthal-1875-1565-1223	1223	glücklich	glücklich	happy	1495	HAPPY
+Steinthal-1875-1565-1224	1224	unglücklich	unglücklich	unhappy	699	SAD
+Steinthal-1875-1565-1225	1225	Glück	Glück	happiness	2941	HAPPINESS
+Steinthal-1875-1565-1226	1226	Unglück	Unglück	misfortune	56	BAD LUCK
+Steinthal-1875-1565-1227	1227	geduldig	geduldig	patient		
+Steinthal-1875-1565-1228	1228	Geduld	Geduld	patience		
+Steinthal-1875-1565-1229	1229	überdrüssig	überdrüssig	weary		
+Steinthal-1875-1565-1230	1230	Ich bin es zufrieden.	Ich bin es zufrieden.	I am satisfied.		
+Steinthal-1875-1565-1231	1231	Ich bin dagegen.	Ich bin dagegen.	I am against it.		
+Steinthal-1875-1565-1232	1232	Das gefällt mir.	Das gefällt mir.	I like it.		
+Steinthal-1875-1565-1233	1233	Gefällt dir das Mädchen?	Gefällt dir das Mädchen?	Do you like the girl?		
+Steinthal-1875-1565-1234	1234	Ich habe dich gern.	Ich habe dich gern.	I like you.		
+Steinthal-1875-1565-1235	1235	Nach deinem Belieben.	Nach deinem Belieben.	As you like it.		
+Steinthal-1875-1565-1236	1236	Ich ziehe dieses vor, es ist besser als jenes.	Ich ziehe dieses vor, es ist besser als jenes.	I prefer this, it is better than that.		
+Steinthal-1875-1565-1237	1237	Ich freue mich, es freut mich.	Ich freue mich, es freut mich.	I‘m pleased.		
+Steinthal-1875-1565-1238	1238	Ich habe genug.	Ich habe genug.	I have enough.		
+Steinthal-1875-1565-1239	1239	Warum bist du so traurig?	Warum bist du so traurig?	Why are you so sad?		
+Steinthal-1875-1565-1240	1240	Was weinst du?	Was weinst du?	Why do you cry?		
+Steinthal-1875-1565-1241	1241	Kummer	Kummer	grief	1668	GRIEF
+Steinthal-1875-1565-1242	1242	Sorge	Sorge	sorrow	1668	GRIEF
```

Comment:
Double mapping

Reply:
IMO these are indeed synonymous so a double mapping makes sense if we want to keep them.